src/Azs.Backup.Admin/examples/Invoke-AzsPruneBackupLocationExternalStore.md

```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
+```
+
+{{ Add description here }}
+
+### Example 2: {{ Add title here }}
+```powershell
```

Comment:
@poetw once you add examples here, if you build again, it should update the docs folder contents to have the examples added